components/scream/src/share/tests/common_physics_functions_tests.cpp

```diff
@@ -112,7 +112,8 @@ void run(std::mt19937_64& engine)
           dz("dz",num_mid_packs),
           z_int("z_int",num_int_packs),
           vmr("volume_mixing_ratio",num_mid_packs),
-          mmr("volume_mixing_ratio",num_mid_packs);
+          mmr("volume_mixing_ratio",num_mid_packs),
```

Comment:
should this read `"mass_mixing_ratio"`?

Reply:
yes, good catch.  Actually Balwinder caught this and changed it in his PR that just went in.  But I think this means there will be conflict, so I'll rebase and repush the branch.